### DISCUSSION TITLE
Searching and highlighting a paragraph

### DISCUSSION THREAD

**Discussion by dineshzende** [pymupdf/PyMuPDF#1815] [Looking for help] [answered]
Hello Authors!
**Let me first thank your wonderful creation. It has helped us and reduced our efforts.**
I am new to the usage of this library. I am actually looking for searching a paragraph and highlighting it. But I can only be able to search and highlight the text on one line. If the search text is spread over multiple lines, it is not showing the results.
- I tried inserting '\n' and '\r' in the search string, but it didn't work.
Can you please help me in understanding how to search text spread on multiple lines (paragraph) ?

**Comment by JorjMcKie** [comment, marked as answer]
answered in #1817
